Content.Shared/Medical/Blood/Components/VascularSystemComponent.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Content.Shared.FixedPoint;
+using Content.Shared.Medical.Blood.Prototypes;
+using Content.Shared.Medical.Blood.Systems;
+using Robust.Shared.GameStates;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Medical.Blood.Components;
+
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState]
+public sealed partial class VascularSystemComponent : Component
+{
+
+    #region Simulation
+
+    /// <summary>
+    /// The healthy volume for this bloodstream.
+    /// </summary>
+    [DataField, AutoNetworkedField] //TODO: Required
+    public FixedPoint2 HealthyVolume = 250;
+
+    /// <summary>
+    /// Healthy blood pressure values.
+    /// The first value is: high (systolic) pressure. This is the blood pressure at the moment blood is pumped.
+    /// The second value is: low (diastolic) pressure. This is the blood pressure in between pumps.
+    /// </summary>
+    [DataField, AutoNetworkedField] //TODO: Required
+    public BloodPressure HealthyBloodPressure = (120,0);
```

Comment:
Paired values should be 120/80? Unless I'm misunderstanding, I read diastole as 0 here.

Reply:
Huh not sure why that's 0, it should be 80 yeah. Nice catch!